wordpress-mobile/WordPress-Android
# Gutenberg - Testing image size options

This Draft PR is for testing Image Size Options, see https://github.com/wordpress-mobile/gutenberg-mobile/pull/1904

### To test:
Add an Image Block and upload an image
Click the gear for block settings
Tap Image Size row and verify value cycles through available sizes (Thumbnail, Medium, Large, Fullscreen).
Verify the Image size changes as you click through the menu.
Verify the Image Size is preserved if you update the post, leave, and return.
PR submission checklist:
